source/js/plugins/manipulation.js

```diff
@@ -300,10 +338,29 @@ export default class ManipulationPlugin
         this._tools.appendChild(threshDiv);
         this._tools.appendChild(sharpDiv);
         this._tools.appendChild(hueDiv);
+        this._tools.appendChild(resetButton);
+        this._tools.appendChild(filterLog);
+
+        this._tools.setAttribute('style', 'padding: 0 1em;');
+    }
+
+    _resetSliders()
+    {
+        // exclude the 3 buttons and filter log
+        for (let i = 2, len = this._tools.children.length - 2; i < len; i++)
```

Comment:
This seems fragile. Can you not do a selector on the elements you want to reset and do it that way? What happens if the next programmer comes along and adds another filter?

Reply:
done